python/thunder/rdds/matrices.py

```diff
@@ -110,7 +110,7 @@ def addInPlace(self, val1, val2):
                 return val1
 
         if method is "reduce":
-            return self.rdd.map(lambda (k, v): v).mapPartitions(matrixsum_iterator_self).sum()
+            return self.rdd.map(lambda (k, v): v).mapPartitions(matrixSum_Iterator_self).sum()
```

Comment:
Here I think one of the two underscores is actually useless, and I've gotten rid of it in the commit that I'm about to post, but I'm leaving it as "matrixSumIterator_self" (and "matrixSumIterator_other"), as these are the "self" and "other" varieties of a "matrixSumIterator".